src/cbuild/core/template.py

```diff
@@ -2125,6 +2137,7 @@ def take_progs(self, man="18"):
         self.take("usr/share/zsh", missing_ok=True)
         self.take("usr/share/fish/completions", missing_ok=True)
         self.take("usr/share/fish/vendor_completions.d", missing_ok=True)
+        self.take("usr/share/nushell/vendor/autoload", missing_ok=True)
```

Comment:
This bit specifically I don't quite understand: Why are we taking completions in `take_progs`, but also in the various completions packages?

Reply:
the completions packages are automatic, i.e. if `foo-progs` takes them, you will then get a `foo-progs-nucomp` package

Reply:
that said, i don't think anything at all right now even provides these?

Reply:
ahh, okay, that makes sense. Thanks :)

Reply:
> that said, i don't think anything at all right now even provides these?

No, nothing ships nushell completions at the moment, but that's probably due to there not being a place to put them yet. Completions themselves do exist already. The previous way to use nushell completions was manually loading every single one of them from your config file, and that didn't even support globbing or anything like that, you manually had to list every single one of them.

I've included a patch here from upstream that's not released yet but already merged, which supports loading these from a system directory.